tools/yaml-templates/ios-test.yml

```diff
@@ -0,0 +1,91 @@
+parameters:
+  - name: 'iOSAppHostingSdkGitPath'
+    default: none
+    type: string
+
+steps:
+  - checkout: self
+  # branch should be changed after test
```

Comment:
Can you say more about what these means?

Reply:
I am not sure what exactly "these" refers to but I would try my best to explain what these codes are doing.

Checkout is a term in yml file to tell pipeline which repo should be checked out. For iOS-related job in the pipeline, we need to check out the Teams-js library repository resource and iOS host SDK repository resource. Self refers to Teams-js library repo.

As for the branch in line 8&9 (including comments), It is a branch in iOS host SDK. Currently there is another PR in SDK side. That is a PR providing some bash scripts  which need to be used for this pipeline. Once we feel this PR is good for testing and publishing results, I would first check in the PR in iOS host SDK side and then change the branch in line 9 to main in this yml file. After that, as this PR be checked in, we then could announce we have a full-connected pipeline.

Hope that makes sense. Please feel free to leave any questions if I didn't explain them clearly.

Reference: https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/steps-checkout?view=azure-pipelines

Reply:
Will think about how to explain it more before I ask Trevor for review.